cmd/hooks/prestop.go

```diff
@@ -37,17 +40,18 @@ func PreStop(clientset kubernetes.Interface) error {
 	}
 
 	node, err := fetchNode(clientset, nodeName)
-	if err != nil {
+	if k8serrors.IsNotFound(err) {
+		klog.InfoS("PreStop: node does not exist, assuming this is a termination event", "node", nodeName)
```

Comment:
Consider adding "and checking for remaining VolumeAttachments" to match line 51 log (because it is not clear to me without looking at code that this is the case. 

e.g.
```
klog.InfoS("PreStop: node does not exist, assuming this is a termination event and checking for remaining VolumeAttachments", "node", nodeName)
```